src/routes/docs/advanced/platform/roles/+page.markdoc

```diff
@@ -0,0 +1,287 @@
+---
+layout: article
+title: Roles
+description: Learn how to setup RBAC roles for the Appwrite Console 
+---
+
+The Appwrite Console supports granular permissions to improve team collaboration and security. Each member of your Console team can be assigned a specific role that grants them access to certain areas of your organization's projects. Below is a breakdown of the new roles available, detailing their permissions and intended use cases.
```

Comment:
Should we mention this is only available on the Scale plan?

Reply:
Roles are available to Pro and Scale . Custom Roles are available for Scale and Enterprise